src/docfx/localization/LocalizationUtility.cs

```diff
@@ -248,6 +248,12 @@ private static string GetLocalizationBranch(LocalizationMapping mapping, string
                 return sourceBranch;
             }
 
+            if (TryRemoveLocale(sourceBranch, out string sourceBranchWithoutLocale, out string branchLocale) &&
```

Comment:
why need change that? For contribution url, we already always get contribution branch, check [here](https://github.com/dotnet/docfx/blob/55900c0b8ffd345adbb7271d17800c137c2f7b34/src/docfx/build/metadata/ContributionProvider.cs#L210)